src/MercadoPago/Client/Invoice/InvoiceClient.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace MercadoPago\Client\Invoice;
+
+use MercadoPago\Client\MercadoPagoClient;
+use MercadoPago\Core\MPRequestOptions;
+use MercadoPago\Exceptions\MPApiException;
+use MercadoPago\MercadoPagoConfig;
+use MercadoPago\Net\HttpMethod;
+use MercadoPago\Net\MPSearchRequest;
+use MercadoPago\Resources\Invoice;
+use MercadoPago\Resources\InvoiceSearch;
+use MercadoPago\Serialization\Serializer;
+
+/** Client responsible for performing invoice actions. */
+class InvoiceClient extends MercadoPagoClient
```

Comment:
```suggestion
final class InvoiceClient extends MercadoPagoClient
```

Reply:
First of all thank you for contributing to this PR, your contribution is very important to us! Since the idea of these classes is not to be a subclass of any other, it does make sense. Good idea!

Reply:
The rationale behind making these classes final is to keep the API as closed an maintainable as possible. Otherwise, users consuming this package can extend these classes, making any change here impossible without forcing us to release a new major version, because the BC promise.
In case we need to expose an extension point, we should use interfaces or abstract classes for that.